cdap-ui/app/cdap/cdap.html

```diff
@@ -31,5 +31,6 @@
   <script type="text/javascript" src="/ui-config.js"></script>
   <script type="text/javascript" src="/cdap_assets/common.js"></script>
   <script type="text/javascript" src="/cdap_assets/cdap.js"></script>
+  <script src="http://localhost:35729/livereload.js"></script>
```

Comment:
Have filed a JIRA for it (https://issues.cask.co/browse/CDAP-7552). Will take it up in subsequent PRs.